packages/components/src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from "react";
+import type { SystemProp, Theme } from "@xstyled/styled-components";
+import { FileWithPath, useDropzone } from "react-dropzone";
+import forEach from "lodash/forEach";
+import { Text } from "../../primitives/Text";
+import { ProgressBar } from "../ProgressBar";
+import { Box } from "../../primitives/Box";
+import { Icon, IconProps } from "../Icon";
+import { CancelUploadButton } from "./CancelUploadButton";
+
+const MAX_FILE_SIZE = 50 * 1024 * 1024;
+
+type DropzoneProps = {
+  /* These props are being saved for V2 */
+  /** The maximum amount allowed for file size. In the format of number?. */
+  // MaxFileSize: number;
+  /** Are multiple file uploads allowed? Boolean */
+  // MultipleFiles: boolean;
+  /** This is the function that gets triggered when a file is dropped */
+  sendDocument: (
+    file: FileWithPath,
+    onProgress?: (progressPercentage: number) => void
+  ) => Promise<unknown>;
+  /** This is the function that gets triggered when a file is dropped */
+  cancelDocumentUpload: (name: string) => Promise<unknown>;
+  /** The file types allowed. In the format of an object. */
+  fileTypes?: Record<string, string[]>;
+};
+/**/
+
+const getVariants = (
```

Comment:
I don't think we have variants for dropzone. Maybe would be better to call it `getColors`.